com.zquestclassic.ZQuest.yml

```diff
@@ -0,0 +1,113 @@
+app-id: com.zquestclassic.ZQuest
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.gnome.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '44'
+finish-args:
+  - --device=all
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-data/ZQuest Classic:create
```

Comment:
Is it for backwards compatibility? Applications have access to $XDG_DATA_HOME by default, as long as they actually use that variable.